docs/src/installing-deis/installing-the-deis-platform.md

```diff
@@ -22,54 +24,65 @@ properly by running
 
 If you see a list of targets like the one above, helm can communicate with the kubernetes master.
 
-Once finished, run this command to provision the Deis platform:
+## Get the Helm Chart
 
-    $ helm update
-    $ helm repo add deis https://github.com/deis/charts
-    $ helm install deis/deis
+The [Helm Deis Chart](https://github.com/deis/charts) contains everything you
+need to install Deis onto your Kubernetes cluster, with a single `helm install` command.
 
-You can then monitor their status by running
+Run the following commands to set up your Helm environment and install the chart:
 
 ```
-$ kubectl get pods --namespace=deis
+$ helm update
+$ helm repo add deis https://github.com/deis/charts
+$ helm fetch deis/deis
 ```
 
-Once you see all of the pods ready, your Deis platform is running on a cluster!
+## Set up the Deis Router
 
-## Routing Traffic
+Now that you have the Deis chart, you'll need to make a minor edit to the
+`deis-router-rc.yaml` Kubernetes manifest file to prepare the router to accept
+incoming traffic on a domain of your choice. Follow the following steps to make
+the edit:
 
-The Deis router component will use an incoming request's HTTP `Host`
-header to direct traffic to Deis components and to applications deployed
-via Deis. It is therefore important that the router be configured
-properly and that DNS and/or your local `/etc/hosts` file are as well.
+1. Open the `$HELM_HOME/workspace/charts/deis/manifests/deis-router-rc.yaml`
+file in your favorite editor. Note that if `HELM_HOME` is not in your environment,
+it defaults to `~/.helm`
```

Comment:
At runtime, the helm tool may consider its home to be `~/.helm` if the `HELM_HOME` env var is not set, but that assumption on the part of the tool isn't applicable to the imperative to "Open the `$HELM_HOME/workspace/charts/deis/manifests/deis-router-rc.yaml` file in your favorite editor."

It would, however, work if it read: "Open the `$(helm home)/workspace/charts/deis/manifests/deis-router-rc.yaml` file in your favorite editor."